mmv1/third_party/terraform/resources/resource_storage_bucket_object.go

```diff
@@ -161,16 +163,52 @@ func resourceStorageBucketObject() *schema.Resource {
 				DiffSuppressFunc: compareCryptoKeyVersions,
 				Description:      `Resource name of the Cloud KMS key that will be used to encrypt the object. Overrides the object metadata's kmsKeyName value, if any.`,
 			},
+
+			"customer_encryption": {
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Sensitive:   true,
```

Comment:
nit, can you check if this conflicts with `kms_key_name`. Try setting both. I imagine this is something we wouldn't want to do unless I am missing something.
```suggestion
				Sensitive:   true,
				ConflictsWith: []string{"kms_key_name"},
```

Reply:
true, that wouldn't make sense, done